maquetado/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+ <title>HTML & CSS Training</title>
+ <meta name="description" content="training">
+ <meta name="author" content="wolox">
+ <link rel="stylesheet" type="text/css" href="scss/out/application.css">
```

Comment:
Actually, the idea is that your `.css` should be in the same folder level as your `index.html`. This works too, but it will be easier for you to have it at the same level everything you start Sass on your project. Also, it's a bit off to set a `css` inside of a `scss` folder